modules/mono/glue/GodotSharp/GodotSharp/Core/Aabb.cs

```diff
@@ -509,6 +509,62 @@ public readonly bool IntersectsPlane(Plane plane)
             return under && over;
         }
 
+        /// <summary>
+        /// Returns <see langword="true"/> if the <see cref="Aabb"/> intersects
+        /// the ray along <paramref name="dir"/> positioned at <paramref name="from"/>.
+        /// </summary>
+        /// <param name="origin">The origin of the ray.</param>
+        /// <param name="dir">The direction of the ray.</param>
+        /// <returns>
+        /// A <see langword="bool"/> for whether or not the <see cref="Aabb"/> intersects the ray.
+        /// </returns>
+        public readonly bool IntersectsRay(Vector3 from, Vector3 dir)
+        {
+            if (HasPoint(from)) return true;
```

Comment:
I don't see this in the Core implementation, and seems a bit redundant.

```suggestion
            if (HasPoint(from))
            {
                return true;
            }
```

Reply:
You're saying remove it? Seems like a potential optimisation to me but I may be wrong

Reply:
We'd have to benchmark it to see if makes a difference.